flycheck.el

```diff
@@ -6972,6 +6973,59 @@ See URL `http://stylelint.io/'."
   :error-parser flycheck-parse-stylelint
   :modes (css-mode))
 
+(flycheck-def-option-var flycheck-cuda-language-standard nil cuda-nvcc
+  "Our CUDA Language Standard "
+  :type '(choice (const :tag "Default standard" nil)
+                 (string :tag "Language standard"))
+  :safe #'stringp
+  :package-version '(flycheck . "0.32"))
```

Comment:
The version number should just be `"32"`, not `"0.32"`.  We have eschewed the dot notation some releases ago.

```suggestion
  :package-version '(flycheck . "32"))
```